packages/react/src/primitives/Expander/Expander.tsx

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import { Root } from '@radix-ui/react-accordion';
+import * as Accordion from '@radix-ui/react-accordion';
```

Comment:
Have to use name space for `radix` components because it does not support Node ESM and so Node will reference its CJS build instead(Node does not know `module: '.../dist/...'). Name space imports will work in both ESM and CJS.

Reply:
Another reason to get rid of radix when we can...

Reply:
Totally agree